src/lib/formats/flopimg.h

```diff
@@ -157,6 +157,8 @@ LEGACY_FLOPPY_OPTIONS_EXTERN(default);
 #define INTERLEAVE(range)       "I" #range
 #define FIRST_SECTOR_ID(range)  "F" #range
 
+/* Max number of excess tracks to be discarded from disk image to fit floppy drive */
+#define DUMP_THRESHOLD 2
```

Comment:
This is a non-descriptive macro that ends up in the global namespace for any drivers that use floppy image formats.  We've already got too many of those.  If it really needs to be in the global namespace, something like `FLOPIMG_MAX_EXCESS_TRACKS` or something would be more descriptive and less likely to cause a collision.